onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -834,11 +834,15 @@ class PlannerImpl {
           return ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "AllocateInputsContiguously() requires all inputs to be initializers, or all inputs to be non-initializers.");
 
         if (actual_plan.alloc_kind == AllocKind::kAllocateStatically) {
-          if (std::find(initializer_allocation_order.begin(), initializer_allocation_order.end(), actual_idx) == initializer_allocation_order.end())
-            initializer_allocation_order.push_back(actual_idx);
+          
```

Comment:
(super nit):  remove extraneous whitespace (here and in `else` block)

Reply:
@jessebenson  Done.